ccd-definition/CaseEventToComplexTypes/CreateClaim.json

```diff
@@ -3,18 +3,10 @@
     "ID": "CourtLocation",
     "CaseEventID": "CREATE_CLAIM",
     "CaseFieldID": "courtLocation",
-    "ListElementCode": "applicantPreferredCourt",
```

Comment:
@sherlynkhaw have you regression tested this locally with an old case.. if you generate one on the old config. Then upload the one with your changes, is the old case able to progress - I believe removing / changing the element codes within lists is a breaking change within CCD

Reply:
@HarryH96 would this be a problem because cases would never go through create case journey again?
also this is just to show the dynamic list. in the service code `applicantPreferredCourt` is then filled in with the 3 digit code attached to the court!